packages/studio/src/components/PageSettingsButton/EntityPageModal.tsx

```diff
@@ -9,13 +9,33 @@ import StreamScopeParser, {
 } from "../../utils/StreamScopeParser";
 import { PageSettingsModalProps } from "./PageSettingsButton";
 import { StaticPageSettings } from "./StaticPageModal";
-import { streamScopeFormData } from "../AddPageButton/StreamScopeCollector";
 import PageDataValidator from "../../utils/PageDataValidator";
 import { toast } from "react-toastify";
 import { isEqual } from "lodash";
 
 type EntityPageSettings = StaticPageSettings & StreamScopeForm;
 
+// TODO (SLAP-2918): Update modal to have stream scope dropdowns
+export const streamScopeFormData: FormData<StreamScopeForm> = {
```

Comment:
It doesn't look like this variable is used in this file so maybe it should live somewhere else?

Reply:
It's used on line 67. We used to import it before, but now it's more specific to this file, so I moved it here